lag/lag.yaml

```diff
@@ -56,9 +82,52 @@ components:
           maximum: 65535
```

Comment:
is this reasonable max/default?

Reply:
This one is an arbitrary identifier - could be any value in the range, even for one lag.

Reply:
And this attribute is moved from per Lag-member level to per Lag. No change in the existing limit.